xc/device.ml

```diff
@@ -1950,3 +1950,66 @@ let get_tc_port ~xs domid =
 	if qemu_exists
 	then Dm.get_tc_port ~xs domid
 	else PV_Vnc.get_tc_port ~xs domid
+
+let qmp_event_tbl = Hashtbl.create 1000
```

Comment:
What is the difference between `qmp_event` and `event_qmp` and what is the purpose of these two tables? Is there a design doc?